src/Writers/HTMLWriter.jl

```diff
@@ -1238,7 +1238,7 @@ function domify(ctx, navnode, contents::Documents.ContentsNode)
         header = anchor.object
         url = string(path, '#', anchor.id, '-', anchor.nth)
         node = a[:href=>url](mdconvert(header.text; droplinks=true))
-        level = Utilities.header_level(header)
+        level = Utilities.header_level(header) - contents_header_level_offset(contents)
```

Comment:
This currently errors with

```
ERROR: LoadError: UndefVarError: contents_header_level_offset not defined
```

Just needs a `using` for it, as the definition is in the `Documenter.Documents` module.